.eslintrc

```diff
@@ -12,7 +12,8 @@
         "eslint:recommended"
     ],
     "ignorePatterns": [
-        "node_modules/"
+        "node_modules/",
+        "database/data/"
```

Comment:
@john-ghatas this was done on the wrong commit.

Reply:
Should be good now

Reply:
this should be merged with the first commit, also please communicate with @MauritsioRK before force pushing as you lost a commit from him.

Reply:
Yes I caught that, Maurits knows we're working on recovering it.